docs/core/app-types.md

```diff
@@ -138,3 +140,10 @@ The following `project.json` sample illustrates a simple self-contained applicat
 }
 ```
 
+### Making a smaller self-contained application
+As mentioned, self-contained application will package the entire runtime with the application. The "entire runtime" here 
```

Comment:
I think that the duplication is acceptable. Given that app type and deployment are tightly coupled, you have to decide up front what kind of app you're going to create. 
The [deployment topic](https://docs.microsoft.com/en-us/dotnet/articles/core/deploying/index) covers both portable and self-contained apps. It could be broken into two tutorials, but generally I find it more helpful to have longer topics that can be used both to provide a comprehensive overview as well as step-by-step instructions for a particular deployment type.